pyuploadcare/secure_url.py

```diff
@@ -87,7 +88,7 @@ def _build_signature(self, expire: int, acl: str) -> str:
         ]
 
         signature = hmac.new(
-            self.secret_key.encode(),
+            binascii.a2b_hex(self.secret_key.encode()),
```

Comment:
Can't find any links to reference or docs.
No info here https://uploadcare.com/docs/security/secure_delivery/

Reply:
I want documentation how to get signature. Where I can read that `a2b_hex` is required here?

Reply:
the secret key should be a hexadecimal string:  https://techdocs.akamai.com/property-mgr/docs/auth-token-2-0-ver
`hmac.new` accepts bytes as a key, hence `a2b_hex`.

in [the reference](https://github.com/akamai/EdgeAuth-Token-Python/blob/master/akamai/edgeauth/edgeauth.py#L175-L179) Akamai code they are doing a similar thing